### PR TITLE
Get stage1 running on OpenBSD

### PR DESCRIPTION
**Pull request by CyberTailor**

### CODE REVIEW

File: scripts/bootstrap-prefix.sh [hunk 1]
```diff
@@ -953,8 +953,9 @@ bootstrap_gnu() {
 	# contain cruft, bug #650284
```
Comment (thesamesam) [inline scripts/bootstrap-prefix.sh:953]: Commit message nit: Andreas Schwab is not the Bash maintainer. Just say you're quoting him from the bash-bug ML and give a link.

**Review by grobian — APPROVED** [review]

**Review by grobian — APPROVED** [review]

**Review by grobian — APPROVED** [review]

**Review by grobian — APPROVED** [review]

File: scripts/bootstrap-prefix.sh [hunk 2]
```diff
@@ -953,8 +953,9 @@ bootstrap_gnu() {
...
 	[[ ${PN} == "bash" ]] && \
 		export CPPFLAGS="${CPPFLAGS} \
-			-DSYS_BASHRC=\'\"${ROOT}/etc/bash/bashrc\"\' \
-			-DSYS_BASH_LOGOUT=\'\"${ROOT}/etc/bash/bash_logout\"\'"
+			-DSYS_BASHRC=\\\"${ROOT}/etc/bash/bashrc\\\" \
```
Comment (grobian) [inline scripts/bootstrap-prefix.sh:956]: and this didn't work before because of the interpreter or something?

**Review by grobian — APPROVED** [review]

**Review by grobian — APPROVED** [review]

**Comment by mgorny** [comment]
@grobian, approval is global, not per-commit.

**Comment by grobian** [comment]
oh crap :( I wanted to approve per commit

**Comment by thesamesam** [comment]
Thanks grobian!=

File: scripts/bootstrap-prefix.sh (hunk 2 above)
Comment (thesamesam) [reply scripts/bootstrap-prefix.sh:956]: Yeah, it was bizarre, but it was something like it'd segfault when it tried to actually access the bashrc path: https://lists.gnu.org/archive/html/bug-bash/2022-06/msg00015.html

File: scripts/bootstrap-prefix.sh (hunk 1 above)
Comment (CyberTailor) [reply scripts/bootstrap-prefix.sh:953]: Changed

**Comment by thesamesam** [comment]
Nice work, thanks!

**Pull request closed without merging**